services/bin/bambi_service

```diff
@@ -15,17 +15,19 @@ LOCAL=
 NO_CHECK=
 [[ ${1:-} == "--no-check" ]] && NO_CHECK=1 && shift
 
+# shellcheck disable=SC2155
+export GCP_PROJECT=$(gcloud config get-value project)
+echo Using GCP_PROJECT "$GCP_PROJECT"
+
 if [[ -n ${UDMI_NAMESPACE:-} ]]; then
     UDMI_PREFIX="${UDMI_NAMESPACE}~"
+    TARGET="//gbos/${GCP_PROJECT}/${UDMI_NAMESPACE}"
```

Comment:
Hmmm... this might be a problem since a lot of things don't use `//gbos/` and I'm not sure we can/should hard-code it.